css/index.css

```
@@ -773,4 +772,211 @@ body {
 
 .food-spin__location-indicator{
   stroke: var(--title-color);
-}
\ No newline at end of file
+}
+
+.cart-summary {
+  max-width: 500px;
+  background-color: var(--color-white);
+  box-shadow: 0 2px 10px rgba(0, 0, 0, 0.1);
+  position: fixed;
+  top: 0%;
+  right: 0;
+  z-index: 3;
+  height: 100%;
+  border: none;
+  padding: 0px;
+  --text-color: #FF922C;
+  --button-color: #FF6600;
+  --box-shopping: rgba(255, 102, 0, 0.25);
+  box-shadow: 0px 0px 70px 20px var(--box-shopping);
+  transform: translateX(200%);
+  transition: transform 0.7s ease, opacity 0.7s ease;
+  opacity: 0;
+}
+
+.cart-summary.cart-summary--active {
```

Que sepa, no hay un `cart-summary--active` dentro de otra clase o con otros estilos
```suggestion
.cart-summary--active {
```